bblfsh/test.py

```diff
@@ -0,0 +1,35 @@
+import socket
+import subprocess
+import time
+import unittest
+
+from bblfsh import BblfshClient
+
+
+class BblfshTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        subprocess.check_call(
+            "docker run --privileged -p 9432:9432 --name bblfsh -d "
```

Comment:
Ideally we would have to mock the server to avoid this external dependencies on the test, but this should serve for now.